orchagent/portsorch.cpp

```diff
@@ -2228,7 +2225,7 @@ void PortsOrch::doPortTask(Consumer &consumer)
                 /* Set autoneg and ignore the port speed setting */
                 else if (fvField(i) == "autoneg")
                 {
-                    an = (int)stoul(fvValue(i));
+                    an = fvValue(i);
```

Comment:
we can change this as an = (fvValue(i) == "on")

then, we do not need to change an throughout the code.

Reply:
done, please check